suite-native/module-accounts-import/src/screens/AccountsImportScreen.tsx

```diff
@@ -44,15 +49,18 @@ export const AccountsImportScreen = ({
     AccountsImportStackRoutes.AccountImport,
     RootStackParamList
 >) => {
+    const { xpubAddress, currencySymbol } = route.params;
     const dispatch = useDispatch();
+    const deviceNetworkAccounts = useSelector(
+        selectDeviceNetworkAccounts(getMockedDeviceState(HIDDEN_DEVICE_ID), currencySymbol),
+    );
     const [accountInfo, setAccountInfo] = useState<AccountInfo | null>(null);
-    const [selectedDevice, setSelectedDevice] = useState<DummyDevice>();
-    const [assetName, setAssetName] = useState<string>('bitcoines #1');
+    const [accountName, setAccountName] = useState<string>(
+        `${currencySymbol} #${deviceNetworkAccounts.length + 1}`,
+    );
```

Comment:
Why not use the slice that you created for state management of account name?

Reply:
Also I did you have a look at suite if there is some basic validation for account name? How many characters, special symbols, etc?

Reply:
> Also I did you have a look at suite if there is some basic validation for account name? How many characters, special symbols, etc?

good point, I will check it and include some validation there.

Reply:
> Why not use the slice that you created for state management of account name?

what do you mean?

Reply:
You're duplicating state here a little bit. Since you already have a map stored in state of account descriptors with names, you can just initialize them with default values and use actions here to update those values. Hopefully it makes more sense. If not, let's have a call and I'll explain better.

Reply:
You probably mean use value from metadata slice as default value of useState? That makes sense.

Reply:
- Input should be red when empty + disable confirm
- Max length on input (if exceeded -> red outline)

Reply:
Yes, this can be the solution in the case when account metadata (descriptor:name) are stored before user confirms (= he submits a button after he has scanned xpub and see balances...) this scanned account - like he wants to have it stored.